packages/react-components/react-rating-preview/src/components/RatingItem/RatingItem.test.tsx

```diff
@@ -1,10 +1,148 @@
 import * as React from 'react';
 import { render } from '@testing-library/react';
+import { isConformant } from '../../testing/isConformant';
 import { RatingItem } from './RatingItem';
+import { RatingItemProvider } from '../../contexts/RatingItemContext';
+import { RatingItemContextValue } from './RatingItem.types';
+import { StarFilled, StarRegular } from '@fluentui/react-icons';
 
 describe('RatingItem', () => {
-  it('renders a default state', () => {
-    const result = render(<RatingItem>Default RatingItem</RatingItem>);
-    expect(result.container).toMatchSnapshot();
+  isConformant({
+    Component: RatingItem,
+    displayName: 'RatingItem',
+    disabledTests: ['component-has-static-classnames-object'],
+  });
+  it('renders the unselectedFilledIcon slot when the rating item value is more than the context value and the mode is not interactive', () => {
+    const contextValues: RatingItemContextValue = {
+      value: 3,
+      interactive: false,
+      iconFilled: <StarFilled />,
+      iconOutline: <StarRegular />,
+      color: 'neutral',
+      step: 1,
+      size: 'medium',
+    };
+    const result = render(
+      <RatingItemProvider value={contextValues}>
+        <RatingItem value={4} id="test" />
+      </RatingItemProvider>,
+    );
+    const items = result.container.querySelector('#test');
+    const child = items?.querySelector('.fui-RatingItem__unselectedFilledIcon');
```

Comment:
It'd be better to leave these icon rendering tests to the VR tests, rather than unit tests.